packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1079,7 +1079,15 @@ class EditableText extends StatefulWidget {
   final ValueChanged<String>? onSubmitted;
 
   /// {@template flutter.widgets.editableText.onAppPrivateCommand}
-  /// Called when the result of an app private command is received.
+  /// This is used to receive a command from the input method.
+  ///
+  /// Called when the result of [TextInputClient.performPrivateCommand] is
+  /// received.
+  ///
+  /// See also:
+  ///   * [https://developer.android.com/reference/android/view/inputmethod/InputMethodManager#sendAppPrivateCommand],
+  ///     which is the Android documentation for sendAppPrivateCommand, used to
+  ///     receive a command from the input method.
```

Comment:
Maybe emphasize the private nature of this parameter, so most developers can just move along? The android documentation of `appPrivateCommand` made it clear that "this can be used to provide domain-specific features that are only known between certain input methods and their clients".

Reply:
Yeah since 99% of developers will never need this.  I'll emphasize "private".